lab_1/main.py

```diff
@@ -11,7 +11,16 @@ def tokenize(text: str) -> list or None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
-    pass
+    symbols = ['.', ',', '!', '*', '?', '-', '\\', '|', '/', ':', ';', '<', '>', '^', '@', '#', '$', '%', '&', '(',
+                   ')', '[', ']', '{', '}', '1', '2', '3', '4', '5', '6', '7', '8', '9', '0']
+    if type(text) == str:
```

Comment:
There should be a check for the input arguments at the top of the function - after the documentation string. Remove nesting here and extract the check into a separate `if` block earlier.